File: Habitica/src/main/java/com/habitrpg/android/habitica/models/responses/SkillResponse.java
package com.habitrpg.android.habitica.models.responses;

import com.habitrpg.android.habitica.models.tasks.Task;
import com.habitrpg.shared.habitica.models.user.User;

import java.util.List;

public class SkillResponse {

    public User user;
    public List<User> partyMembers;
    public Task task;

    public double expDiff;
    public double hpDiff;
    public double goldDiff;

}

File: Habitica/src/main/java/com/habitrpg/android/habitica/models/responses/SkillResponse.kt
package com.habitrpg.android.habitica.models.responses

import com.habitrpg.android.habitica.models.tasks.Task
import com.habitrpg.android.habitica.models.user.User

class SkillResponse {
    var user: User? = null
    var task: Task? = null
    var expDiff = 0.0
    var hpDiff = 0.0
    var goldDiff = 0.0
    var damage = 0.0f
}